### PR TITLE
Update 2024-09-23-changelog.md

### PR DESCRIPTION
**Pull request by kculala**

### CODE REVIEW

File: changelog/2024/2024-09-23-changelog.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
-**MINOR** We have now introduced a new optional field, `order_number`, which serves as an alternative to the existing `order_id`. For Authorize.net transactions, `order_number` will be sent instead of the `order_id`, and will appear as the invoice number. For more information, see [Update an Order](/docs/api/carts/update-an-order).
\ No newline at end of file
+**MINOR** We have now introduced a new optional field, `order_number`, which serves as an alternative to the existing `order_id`. For Authorize.net transactions, `order_number` will be sent instead of the `order_id`, and will appear as the invoice number. For more information, see [Update an Order](/docs/api/carts/update-an-order).
+
+**MINOR** We have added the ability to configure string Custom Fields for `use_as_url_slug`, this will allow you to use the value of that Custom Field to access your Custom API Entry in a human readable way. For more information, see [Update Custom API Entries](/guides/How-To/commerce-extensions/create-a-multilocation-inventories-resource#update-custom-api-entries).
```
Comment (kculala) [inline changelog/2024/2024-09-23-changelog.md:12]: ok it's not inherently human readable as a sku for example if marked for use is not really human readable but you _could_ have it so.

maybe we should just restate one of the things said here https://elasticpath.dev/guides/How-To/commerce-extensions/create-a-multilocation-inventories-resource#update-custom-api-entries

thoughts?

Comment (rahinisrinivas) [reply changelog/2024/2024-09-23-changelog.md:12]: Can we say something like this:

While a value like a SKU may not be inherently human-readable, but you can configure more descriptive values if needed.

Comment (rahinisrinivas) [inline changelog/2024/2024-09-23-changelog.md:12]:
```suggestion
**MINOR** We have added the ability to configure string Custom Fields for `use_as_url_slug`, allowing you to use the value of that Custom Field to access your Custom API Entry in a human readable way. For more information, see [Update Custom API Entries](/guides/How-To/commerce-extensions/create-a-multilocation-inventories-resource#update-custom-api-entries).
```

Comment (kculala) [reply changelog/2024/2024-09-23-changelog.md]: I ended up pivoting away from human-readable and instead reiterating something said in the linked doc

Comment (rahinisrinivas) [reply changelog/2024/2024-09-23-changelog.md]: ok makes sense.

**Review by rahinisrinivas — APPROVED** [review]

**Pull request merged**